appinventor/blocklyeditor/src/replmgr.js

```diff
@@ -441,6 +441,40 @@ Blockly.ReplMgr.putYail = (function() {
             poll();
 
         },
+        'chunker' : (function() {
+            var seq = 0;
+            var gensym = function() {
+                seq += 1;
+                return 'Q' + seq;
+            };
+
+            var chunker = function(input) {
+                var length = input.length;
+                var chunklen = 8192;
+                if (length < chunklen) {
+                    return input;
```

Comment:
Maybe we should return `[input]` instead so that the return type of this function is always `Array.<string>`. This way, all of the branching statements below collapse.